data_models/data_refinery_models/migrations/0002_auto_20170727_2100.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```

Comment:
It did. And then I realized I needed to make another change midway through the run of all the processors. I didn't want to blow away the DB because at that point there was production data in it, so I just ran the single additional migration.